packages/near-fast-auth-signer/.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 
```

Comment:
Steps to reproduce

1 - Create account on Chrome
2 - Logout from Chrome
2 - Login on Safari
3 - Logout from Safari
4 - Try to login from Safari
5 - Passkeys won't be requested and e-mail will be sent. 

Screen recording

https://drive.google.com/file/d/1zcvuILfF-4-L3zIY3LRCjVX2th6CyOhn/view?usp=sharing

Reply:
@Pessina please open a separate issue for this thanks